### ISSUE TITLE
ESP32 device hang / lagging with emulation enabled after upgrade to 14.0.0

### ISSUE THREAD

**Issue by Mark-Muc** [arendst/Tasmota#21470]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
I do hvbe some 32 Devices (in this case 2 Sonoff SwitchMan 2C) running on Tasmota 13.4.0(TasmoCompiler-esp32generic)  without any problem.
If i update to 14.0.0.1 (selfe compiled) switches will be attached.
If i upgrade fist I have to update twice (1. time it will stay in safeboot, 2. update it works) ..after it shows up ist run quite a while (2-3 min) and then it lagged like hell.
I tried it several time in other ESP32 models . evertime the same.
I fount out, if i do someting on MQTT (disable; delete the server) it hangs asap.
I tried really @ least 10 times . everytime the same!

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_
compileswitches:
![image](https://github.com/arendst/Tasmota/assets/67410847/c40e4f9e-e96a-40b7-b8bf-69332cae5a36)

my config:
```
autoexec.bat 
Mi32Option6 1
MI32Page 10
MI32Block A4C138465A57
MI32Block E7764500F442
BLEAlias 150311116C66=LYWSD02-1 A4C138AF4A0B=Temp&Hum(LYWSD03MMC) E72E0050A0CC=1.OG_Bad A4C138465A57=Pavillion_innen A4C1382D29C3=1.OG_Arbeit_2 A4C1389C4D2A=Thermostat
shutteropen

also attached my config: can provide via mail!!


```

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
```
```
09:26:17.464 CMD: status0
09:26:17.489 RSL: STATUS0 = {"Status":{"Module":0,"DeviceName":"Rollo 1. OG Arbeit 1 Ost","FriendlyName":["Rollo 1. OG Arbeit 1 Ost auf","Rollo 1. OG Arbeit 1 Ost zu","","","",""],"Topic":"Rollo_OG_Arbeit_1_Ost","ButtonTopic":"0","Power":"100000","PowerLock":"000000","PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0},"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"https://ota.tasmota.com/tasmota32/release/tasmota32.bin","RestartReason":"Vbat power on reset","Uptime":"0T00:22:31","StartupUTC":"2024-05-23T07:03:46","Sleep":50,"CfgHolder":4617,"BootCount":10,"BCResetTime":"2023-02-27T13:35:34","SaveCount":28},"StatusFWR":{"Version":"14.0.0.1(TasmoCompiler-esp32generic)","BuildDateTime":"2024.05.23 06:50:08","Core":"3_0_0","SDK":"5.1.3.240430","CpuFrequency":160,"Hardware":"ESP32-D0WDQ6-V3 v3.0","CR":"528/699"},"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Aichacher",""],"TelePeriod":300,"Resolution":"55C180C0","SetOption":["0000E001","2805C80001000600003C5A0A192800000000","40000088","00006000","00004002","00000000"]},"StatusMEM":{"ProgramSize":1524,"Free":1355,"Heap":49,"StackLowMark":3,"PsrMax":0,"PsrFree":0,"ProgramFlashSize":4096,"FlashSize":4096,"FlashChipId":"164054","FlashFrequency":40,"FlashMode":"DIO","Features":["0407","8F8047CB","00140001","00000007","00000000","C0000981","00000080","00200000","5400000C","40000080","00000004"],"Drivers":"1,!2,!3,4,!8,9,10,!12,20,!21,!24,26,27,29,!35,38,50,52,62,!68,79,!121","Sensors":"1,2,3,52,62,127"},"StatusNET":{"Hostname":"Rollo-1OG-Arbeit-1-OST","IPAddress":"192.168.11.96","Gateway":"192.168.11.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.11.1","DNSServer2":"0.0.0.0","Mac":"24:0A:C4:F8:F3:90","IP6Global":"","IP6Local":"fe80::260a:c4ff:fef8:f390%st1","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0},"StatusTIM":{"UTC":"2024-05-23T07:26:17Z","Local":"2024-05-23T09:26:17","StartDST":"2024-03-31T02:00:00","EndDST":"2024-10-27T03:00:00","Timezone":99,"Sunrise":"05:24","Sunset":"20:55"},"StatusSNS":{"Time":"2024-05-23T09:26:17","Shutter1":{"Position":100,"Direction":0,"Target":100,"Tilt":0}},"StatusSTS":{"Time":"2024-05-23T09:26:17","Uptime":"0T00:22:31","UptimeSec":1351,"Heap":48,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":32},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"ON","Dimmer":80,"Fade":"OFF","Speed":1,"LedTable":"ON","Wifi":{"AP":1,"SSId":"Aichacher","BSSId":"98:9B:CB:30:43:B9","Channel":6,"Mode":"HT40","RSSI":100,"Signal":-46,"LinkCount":3,"Downtime":"0T00:00:06"}},"StatusSHT":{"SHT0":{"Relay1":1,"Relay2":2,"Open":212,"Close":201,"50perc":53,"Delay":0,"Opt":"0100","Calib":[260,398,918,900,1000],"Mode":"1","TiltConfig":[0,0,0,0,0]}}}

``````

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
log looks normal:
```
0:00:01.000 WIF: verbinden mit AP1 xyz Channel 6 BSSId 98:9B:CB:30:43:B9 in Modus HT40 als Rollo-1OG-Arbeit-1-OST...
IMPROV 
00:00:03.596 WIF: verbunden
00:00:03.803 HTP: Webserver aktiv bei Rollo-1OG-Arbeit-1-OST mit IP-Adresse 192.168.11.96
00:00:03.807 UPP: Multicast (wieder-)verbunden
00:00:03.810 RSL: INFO1 = {"Info1":{"Module":"Sonoff SwitchMan 2C LED (button)","Version":"14.0.0.1(TasmoCompiler-esp32generic)","FallbackTopic":"cmnd/Rollo_OG_Arbeit_1_Ost_fb/","GroupTopic":"cmnd/tasmotas/"}}
00:00:03.840 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"Rollo-1OG-Arbeit-1-OST","IPAddress":"192.168.11.96","IP6Global":"","IP6Local":"fe80::260a:c4ff:fef8:f390%st1"}}
00:00:03.859 RSL: INFO3 = {"Info3":{"RestartReason":"Vbat power on reset","BootCount":10}}
00:00:03.867 RSL: RESULT = {"POWER1":"OFF"}
00:00:03.868 RSL: POWER1 = OFF
00:00:03.870 RSL: RESULT = {"POWER2":"OFF"}
00:00:03.871 RSL: POWER2 = OFF
00:00:03.872 RSL: RESULT = {"POWER3":"OFF"}
00:00:03.883 RSL: POWER3 = OFF
00:00:03.884 RSL: RESULT = {"POWER4":"OFF"}
00:00:03.885 RSL: POWER4 = OFF
00:00:03.886 RSL: RESULT = {"POWER5":"OFF"}
00:00:03.898 RSL: POWER5 = OFF
00:00:03.899 RSL: RESULT = {"POWER6":"ON"}
00:00:03.900 RSL: POWER6 = ON
09:03:50.002 BLE: MasterEnable->1
09:03:50.031 RSL: RESULT = {"MI32Option6":1}
09:03:50.050 BLE: BLETask:Starting NimBLE
09:03:50.092 RSL: RESULT = {"MI32Page":10}
09:03:50.154 RSL: RESULT = {"MI32Block":{"A4C138465A57":1}}
09:03:50.214 RSL: RESULT = {"MI32Block":{"A4C138465A57":1,"E7764500F442":1}}
09:03:50.272 BLE: Add Alias mac 150311116C66 = name LYWSD02-1
09:03:50.273 BLE: Add Alias mac A4C138AF4A0B = name Temp&Hum(LYWSD03MMC)
09:03:50.273 BLE: Add Alias mac E72E0050A0CC = name 1.OG_Bad
09:03:50.285 BLE: Add Alias mac A4C138465A57 = name Pavillion_innen
09:03:50.286 BLE: Add Alias mac A4C1382D29C3 = name 1.OG_Arbeit_2
09:03:50.297 BLE: Add Alias mac A4C1389C4D2A = name Thermostat
09:03:50.300 RSL: RESULT = {"BLEAlias":{"150311116C66":"LYWSD02-1","A4C138AF4A0B":"Temp&Hum(LYWSD03MMC)","E72E0050A0CC":"1.OG_Bad","A4C138465A57":"Pavillion_innen","A4C1382D29C3":"1.OG_Arbeit_2","A4C1389C4D2A":"Thermostat"}}
09:03:50.377 RSL: RESULT = {"Shutter1":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}
09:03:53.902 QPC: Reset
09:03:53.910 RSL: STATE = {"Time":"2024-05-23T09:03:53","Uptime":"0T00:00:07","UptimeSec":7,"Heap":61,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":32},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"ON","Dimmer":80,"Fade":"OFF","Speed":1,"LedTable":"ON","Wifi":{"AP":1,"SSId":"Aichacher","BSSId":"98:9B:CB:30:43:B9","Channel":6,"Mode":"HT40","RSSI":100,"Signal":-44,"LinkCount":1,"Downtime":"0T00:00:03"}}
09:03:53.957 RSL: SENSOR = {"Time":"2024-05-23T09:03:53","Shutter1":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}
09:03:53.963 RSL: BLE = {"Time":"2024-05-23T09:03:53","BLEDevices":{"total":3,"F877B85FE13D":{"i":0,"r":-70},"001EC0779B33":{"i":1,"n":"D1703174","r":-81},"150311116C66":{"i":2,"r":-51,"a":"LYWSD02-1"}}}
09:03:53.993 RSL: BLE = {"Time":"2024-05-23T09:03:53","BLE":{"scans":0,"adverts":25,"devices":3,"resets":0}}
09:03:54.003 RSL: LYWSD02-1 = {"Time":"2024-05-23T09:03:54","alias":"LYWSD02-1","mac":"150311116c66","Temperature":23.8,"RSSI":-51,"TempUnit":"C"}
09:03:55.929 RSL: 1.OG_Bad = {"Time":"2024-05-23T09:03:55","alias":"1.OG_Bad","mac":"e72e0050a0cc","Battery":27,"RSSI":-92}
09:04:00.930 RSL: 1.OG_Arbeit_2 = {"Time":"2024-05-23T09:04:00","alias":"1.OG_Arbeit_2","mac":"a4c1382d29c3","Temperature":21.9,"Humidity":49.2,"DewPoint":10.7,"Btn":0,"Battery":84,"RSSI":-79,"TempUnit":"C"}
09:07:42.800 UPP: Multicast (wieder-)verbunden
09:08:47.904 RSL: LYWSD02-1 = {"Time":"2024-05-23T09:08:47","alias":"LYWSD02-1","mac":"150311116c66","Temperature":23.8,"Humidity":47.0,"DewPoint":11.8,"Battery":45,"RSSI":-57,"TempUnit":"C"}
09:08:48.904 RSL: 1.OG_Bad = {"Time":"2024-05-23T09:08:48","alias":"1.OG_Bad","mac":"e72e0050a0cc","Temperature":21.4,"Humidity":61.0,"DewPoint":13.6,"Battery":27,"RSSI":-95,"TempUnit":"C"}
09:08:49.904 RSL: 1.OG_Arbeit_2 = {"Time":"2024-05-23T09:08:49","alias":"1.OG_Arbeit_2","mac":"a4c1382d29c3","Temperature":21.9,"Humidity":49.1,"DewPoint":10.7,"Btn":0,"Battery":84,"RSSI":-79,"TempUnit":"C"}
09:08:53.905 RSL: STATE = {"Time":"2024-05-23T09:08:53","Uptime":"0T00:05:07","UptimeSec":307,"Heap":50,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":32},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"ON","Dimmer":80,"Fade":"OFF","Speed":1,"LedTable":"ON","Wifi":{"AP":1,"SSId":"Aichacher","BSSId":"98:9B:CB:30:43:B9","Channel":6,"Mode":"HT40","RSSI":100,"Signal":-44,"LinkCount":2,"Downtime":"0T00:00:04"}}
09:08:53.956 RSL: SENSOR = {"Time":"2024-05-23T09:08:53","Shutter1":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}
09:08:53.965 RSL: BLE = {"Time":"2024-05-23T09:08:53","BLEDevices":{"total":6,"F877B85FE13D":{"i":0,"r":-71},"001EC0779B33":{"i":1,"n":"D1703174","r":-81},"150311116C66":{"i":2,"r":-51,"a":"LYWSD02-1"},"E72E0050A0CC":{"i":3,"r":-95,"a":"1.OG_Bad"},"A4C1382D29C3":{"i":4,"r":-79,"a":"1.OG_Arbeit_2"},"E7764500F442":{"i":5,"r":-94}}}
09:08:53.998 RSL: BLE = {"Time":"2024-05-23T09:08:53","BLE":{"scans":15,"adverts":3302,"devices":6,"resets":0}}
09:13:47.922 RSL: LYWSD02-1 = {"Time":"2024-05-23T09:13:47","alias":"LYWSD02-1","mac":"150311116c66","Temperature":23.8,"Humidity":47.0,"DewPoint":11.8,"Battery":46,"RSSI":-51,"TempUnit":"C"}
09:13:48.923 RSL: 1.OG_Bad = {"Time":"2024-05-23T09:13:48","alias":"1.OG_Bad","mac":"e72e0050a0cc","Temperature":21.4,"Humidity":61.0,"DewPoint":13.6,"Battery":27,"RSSI":-91,"TempUnit":"C"}
09:13:49.923 RSL: 1.OG_Arbeit_2 = {"Time":"2024-05-23T09:13:49","alias":"1.OG_Arbeit_2","mac":"a4c1382d29c3","Temperature":21.9,"Humidity":49.0,"DewPoint":10.6,"Btn":0,"Battery":84,"RSSI":-81,"TempUnit":"C"}
09:13:53.924 RSL: STATE = {"Time":"2024-05-23T09:13:53","Uptime":"0T00:10:07","UptimeSec":607,"Heap":50,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":32},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"ON","Dimmer":80,"Fade":"OFF","Speed":1,"LedTable":"ON","Wifi":{"AP":1,"SSId":"Aichacher","BSSId":"98:9B:CB:30:43:B9","Channel":6,"Mode":"HT40","RSSI":100,"Signal":-45,"LinkCount":2,"Downtime":"0T00:00:04"}}
09:13:53.974 RSL: SENSOR = {"Time":"2024-05-23T09:13:53","Shutter1":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}
09:13:53.982 RSL: BLE = {"Time":"2024-05-23T09:13:53","BLEDevices":{"total":6,"F877B85FE13D":{"i":0,"r":-68},"001EC0779B33":{"i":1,"n":"D1703174","r":-88},"150311116C66":{"i":2,"r":-50,"a":"LYWSD02-1"},"E72E0050A0CC":{"i":3,"r":-91,"a":"1.OG_Bad"},"A4C1382D29C3":{"i":4,"r":-81,"a":"1.OG_Arbeit_2"},"E7764500F442":{"i":5,"r":-95}}}
09:13:54.015 RSL: BLE = {"Time":"2024-05-23T09:13:54","BLE":{"scans":30,"adverts":6755,"devices":6,"resets":0}}
09:17:43.801 UPP: Multicast (wieder-)verbunden
09:18:47.903 RSL: LYWSD02-1 = {"Time":"2024-05-23T09:18:47","alias":"LYWSD02-1","mac":"150311116c66","Temperature":23.8,"Humidity":47.0,"DewPoint":11.8,"Battery":46,"RSSI":-49,"TempUnit":"C"}
09:18:48.902 RSL: 1.OG_Bad = {"Time":"2024-05-23T09:18:48","alias":"1.OG_Bad","mac":"e72e0050a0cc","Temperature":21.4,"Humidity":61.0,"DewPoint":13.6,"Battery":27,"RSSI":-92,"TempUnit":"C"}
09:18:49.902 RSL: 1.OG_Arbeit_2 = {"Time":"2024-05-23T09:18:49","alias":"1.OG_Arbeit_2","mac":"a4c1382d29c3","Temperature":21.9,"Humidity":49.0,"DewPoint":10.7,"Btn":0,"Battery":84,"RSSI":-82,"TempUnit":"C"}
09:18:53.902 RSL: STATE = {"Time":"2024-05-23T09:18:53","Uptime":"0T00:15:07","UptimeSec":907,"Heap":50,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":32},"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"ON","Dimmer":80,"Fade":"OFF","Speed":1,"LedTable":"ON","Wifi":{"AP":1,"SSId":"Aichacher","BSSId":"98:9B:CB:30:43:B9","Channel":6,"Mode":"HT40","RSSI":100,"Signal":-44,"LinkCount":3,"Downtime":"0T00:00:06"}}
09:18:53.952 RSL: SENSOR = {"Time":"2024-05-23T09:18:53","Shutter1":{"Position":100,"Direction":0,"Target":100,"Tilt":0}}
09:18:53.960 RSL: BLE = {"Time":"2024-05-23T09:18:53","BLEDevices":{"total":6,"F877B85FE13D":{"i":0,"r":-70},"001EC0779B33":{"i":1,"n":"D1703174","r":-91},"150311116C66":{"i":2,"r":-57,"a":"LYWSD02-1"},"E72E0050A0CC":{"i":3,"r":-92,"a":"1.OG_Bad"},"A4C1382D29C3":{"i":4,"r":-82,"a":"1.OG_Arbeit_2"},"E7764500F442":{"i":5,"r":-95}}}
09:18:53.992 RSL: BLE = {"Time":"2024-05-23T09:18:53","BLE":{"scans":45,"adverts":10373,"devices":6,"resets":0}}

```>



```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by rschitz** [comment]
Sonoff Zigbee Pro 14.0.0.1(a9e7426-zbbrdgpro) reboot when i go to the /wi page

**Comment by s-hadinger** [comment]
Please don't mix up problems. Lag is one, crash on a specific page is completely different

**Comment by s-hadinger** [comment]
Can you please try `Sleep 0

**Comment by s-hadinger** [comment]
Can you try `Sleep 0`. This is not the final solution but will eliminate some possible problems. 

`Sleep 0` prevents wifi from going in sleep mode and prevents lagging problems related to wifi

**Comment by Mark-Muc** [comment]
nothig changed on `sleep 0`

**Comment by s-hadinger** [comment]
Did you try rebooting your wifi Access Point?

**Comment by ewanfletcher** [comment]
I had the same issue as the OP - Web interface was slow, and would stall completely - I ended up pulling the switch and flashing with an FTDI to get back to 13.4.0

Simialr device - The Sonoff M5-1C-86

_May have had a simialr issue with a couple of very early Sonoff Basics R1s - Haven't had time to retest yet_

**Comment by arendst** [comment]
It is now fixed in v14.0.0.3
